--- healpy-1.8.1.orig/setup.py  2014-06-21 17:45:16.000000000 -0700
+++ healpy-1.8.1/setup.py   2014-10-16 13:36:47.000000000 -0700
@@ -127,8 +127,9 @@
     if e.errno != errno.ENOENT:
         raise ValueError
     log.warn('pkg-config is not installed, falling back to pykg-config')
-    setup_requires = ['pykg-config >= 1.2.0']
-    os.environ['PKG_CONFIG'] = sys.executable + ' ' + os.path.abspath('pykg_config.py')
+    setup_requires = []
+    #This is set up by requiring pykg_config
+    os.environ['PKG_CONFIG'] = os.path.join(os.environ['PYKG_CONFIG_DIR'], 'bin', 'pykg-config.py')


 class build_external_clib(build_clib):
@@ -439,7 +440,6 @@
       ext_modules = extension_list,
       package_data = {'healpy': ['data/*.fits', 'data/totcls.dat', 'test/data/*.fits', 'test/data/*.sh']},
       setup_requires=setup_requires,
-      install_requires=['pyfits', 'six'],
       tests_require=['pytest'],
       test_suite='healpy',
       license='GPLv2'
